ocs_ci/helpers/managed_services.py

```diff
@@ -525,6 +526,14 @@ def verify_faas_provider_resources():
     log.info("Verifying managedFusionOffering onboardingValidationKey")
     assert len(offering_info["spec"]["config"]["onboardingValidationKey"]) > 700
 
+    # Verify that version of Prometheus is 4.10.0
+    prometheus_csv = csv.get_csvs_start_with_prefix(
+        constants.OSE_PROMETHEUS_OPERATOR, config.ENV_DATA["cluster_namespace"]
+    )
+    prometheus_version = prometheus_csv[0]["spec"]["version"]
+    log.info(f"Verifying Prometheus version: {prometheus_version}")
+    assert prometheus_version == "4.10.0"
```

Comment:
Wouldn't it be better to take the version from config file? It can be in defaults

Reply:
Thank you! Moved the version to the default config file